src/test/java/seedu/address/testutil/PocBuilder.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.testutil;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.poc.Email;
+import seedu.address.model.poc.PersonName;
+import seedu.address.model.poc.Phone;
+import seedu.address.model.poc.Poc;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building Poc objects.
+ */
+public class PocBuilder {
+
+    public static final String DEFAULT_NAME = "Amy Bee";
+    public static final String DEFAULT_PHONE = "85355255";
+    public static final String DEFAULT_EMAIL = "amy@gmail.com";
+    public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
```

Comment:
I think address is no longer needed in PocBuilder considering a POC does not have an address